src/taipy/core/data/excel.py

```diff
@@ -80,6 +80,7 @@ class ExcelDataNode(DataNode, _AbstractFileDataNode, _AbstractTabularDataNode):
     __DEFAULT_PATH_KEY = "default_path"
     __HAS_HEADER_PROPERTY = "has_header"
     __SHEET_NAME_PROPERTY = "sheet_name"
+    __PANDAS_DEFAULT_SHEET_NAME = "Sheet1"
```

Comment:
Should we clarify this in the doc, so the user know that the default value is?

Reply:
as discussed, we will revert to the previous version :D